### PR TITLE
Remove Sentry

### PR DESCRIPTION
**Pull request by webb04**
We can put this in a conditional that's populated by the native layer.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -11919,7 +11862,8 @@
     "tslib": {
       "version": "1.9.3",
       "resolved": "https://registry.npmjs.org/tslib/-/tslib-1.9.3.tgz",
-      "integrity": "sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ=="
+      "integrity": "sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ==",
+      "dev": true
```
Comment (jorgeazevedo) [inline package-lock.json]: hm is this dev property supposed to be here?

**Comment by jorgeazevedo** [comment]
Yay! I think this will be very easy to revert and re-enable later on

File: package-lock.json (hunk 1 above)
Comment (webb04) [reply package-lock.json]: Should be ok, just a result of removing sentry I guess!
https://docs.npmjs.com/files/package-lock.json#dev

Comment (jorgeazevedo) [reply package-lock.json]: cool!

**Comment by SiAdcock** [comment]
Have you discussed with InfoSec whether tomhis is even necessary? I can put you in touch with some people who I’m sure would be more than happy to look into this, as they’ve done so with the website in the past

**Pull request merged**